_source/_posts/2019-09-27-build-a-crud-app-with-jsf.md

```diff
@@ -0,0 +1,786 @@
+---
+layout: blog_post
+title: "Build a Simple Application with JavaServer Faces"
```

Comment:
Should CRUD be in the title? It's in the URL and description. It looks like the keywords are `java jsf` and neither is in the title. `Build a CRUD App with Java and JSF` is a 57 on https://brentisarulebreaker.dev/